src/easydmp/dmpt/models.py

```diff
@@ -679,9 +679,31 @@ def _get_last_answered_on_trunk_question(self, answers):
 
     # END: Section movement helpers
 
+    def get_optional_section_question(self):
+        if self.optional:
+            return self.questions.get(position=0).get_instance()
+        return None
+
+    def is_skipped(self, data):
+        if not self.optional:  # Non-optional sections can never be skipped
+            return False
+        toggle_question = self.get_optional_section_question()
+        toggle_answer = toggle_question.get_answer_choice(data)
+        if not data or not toggle_answer:
+            return True  # Should simplify logic elsewhere
+        toggle = toggle_question._serialize_condition(toggle_answer)
+        if toggle == 'Yes':
+            return True
+        return False
+
     def generate_canned_text(self, data):
```

Comment:
Ok, now I finally understand what we mean with canned text in this model, it's like text export.